5calls/app/src/main/java/org/a5calls/android/a5calls/controller/StatsActivity.java

```diff
@@ -143,13 +157,15 @@ private void initializeUI(DatabaseHelper db) {
         }
         */
 
-        createGraph(contacts, voicemails, unavailables);
+        createPieChart(contacts, voicemails, unavailables);
+        createLineGraph(contacts, voicemails, unavailables);
 
         // Show the share button.
         invalidateOptionsMenu();
     }
 
-    private void createGraph(List<Long> contacts, List<Long> voicemails, List<Long> unavailables) {
+    private void createPieChart(List<Long> contacts, List<Long> voicemails, List<Long> unavailables) {
+        //find first time when user made any call
```

Comment:
nit: please format comments by starting with a space after the //, and writing as a sentence (beginning with a capital letter and ending with a period).

thanks!